tests/test_merge.py

```diff
@@ -21,7 +21,7 @@ def setUp(self) -> None:
     def test_merge_multiple_inputs(self):
         """Test merging of multiple msdfs."""
         merged_msdf = merge_msdf(*self.msdfs)
-        self.assertEqual(71, len(merged_msdf.df))
+        self.assertEqual(20, len(merged_msdf.df))
```

Comment:
Why this change?

Reply:
with `match_type` changing to be single valued, duplicate rows get filtered out. So length o dataframes will change.